src/Image/Image.js

```diff
@@ -285,7 +293,8 @@ module.exports = kind(
 	*/
 	handleError: function () {
 		if (this.placeholder) {
-			this.set('_src', null);
+			this.set('src', this._transparentImg);
```

Comment:
We shouldn't overwrite the user's value of `src`. That's why we have the `_src` member.